transforms/mods_to_dc_oai.xsl

```diff
@@ -261,7 +261,7 @@
 			</xsl:when>	
 			<xsl:otherwise>
 				<dc:type>
-					Image
+					image
```

Comment:
Why do we lowercase image here?  Is there a reason that I'm missing?

Reply:
No, that looks like a mistake on my part. I'll look over this and make any needed edits before tomorrow.